pom.xml

```diff
@@ -99,6 +99,10 @@
             <artifactId>testcontainers</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>auth-crt</artifactId>
```

Comment:
Had to manually add since it's not in the `awssdk` `pom` we already use.

Reply:
+1